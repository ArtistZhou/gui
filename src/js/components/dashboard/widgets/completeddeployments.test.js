import React from 'react';
import renderer from 'react-test-renderer';
import CompletedDeployments from './completeddeployments';

it('renders correctly', () => {
  const tree = renderer.create(<CompletedDeployments deployments={[]} />).toJSON();
  expect(tree).toMatchSnapshot();
});
