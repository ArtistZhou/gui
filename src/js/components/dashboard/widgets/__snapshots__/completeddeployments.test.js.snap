// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompletedDeployments Component renders correctly 1`] = `null`;
